### ISSUE TITLE
Rename `CharacteristicSubgroupsLib` into `CharacteristicSubgroups`

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#975]
@hulpke recently added a new attribute `CharacteristicSubgroupsLib` to the library in the master branch. It is named like that to avoid a conflict with `crisp`, which already defines `CharacteristicSubgroups`. But both attributes serve the same purpose, so they should be merged before GAP 4.9.0

I guess the plan should be to simply rename the attribute to `CharacteristicSubgroups` in the GAP library, and coordinate with @bh11 to make a new release of crisp with
```gap
DeclareAttribute("CharacteristicSubgroups", IsGroup);
```
replaced by something like this:
```gap
if not IsBound(CharacteristicSubgroups) then
    DeclareAttribute("CharacteristicSubgroups", IsGroup);
fi;
```

Thoughts?

**Comment by hulpke** [comment]
Yes. I can do the renaming once there is a new version of crisp.

**Comment by bh11** [comment]
Back in #878, we already discussed this and agreed that the best solution would be to change DeclareAttribute to silently ignore duplicate declarations. Seems a much cleaner approach to me than to add a hack to CRISP.

**Comment by fingolfin** [comment]
@bh11 Yeah, I simply forgot about that. Anyway, it's been resolved this way now via #978.
